tests/PolyType.Tests/PolyType.Tests.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  
+
   <PropertyGroup>
-    <TargetFrameworks>net9.0;net8.0;net472</TargetFrameworks>
+    <TargetFrameworks>net9.0;net8.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net472</TargetFrameworks>
```

Comment:
I had enabled x-plat net472 testing intentionally so that it can be tested in non-Windows dev environments and  also to make sure that mono works with polytype.

Reply:
I removed it because mono didn't like the presence of the ModuleInitializer that I added. But I'll workaround it another way so you can keep testing on mono then.